plugins/resources/unixfilesystem/include/irods/private/irods_is_in_host_list.hpp

```diff
@@ -0,0 +1,15 @@
+#ifndef IRODS_IS_IN_HOST_LIST_HPP
+#define IRODS_IS_IN_HOST_LIST_HPP
+
+#include "irods/irods_lookup_table.hpp"
+
+#include <optional>
+#include <string>
+#include <tuple>
+
+auto is_host_in_host_list(
+    irods::plugin_property_map& prop_map,
+    const std::string& resource_hostname,
+    const std::string& resource_name) -> bool;
```

Comment:
This function declaration and its implementation should be placed inside of the `irods` namespace.

Reply:
done